phylogenetic/rules/construct_phylogeny.smk

```diff
@@ -30,6 +30,23 @@ rule tree:
             --nthreads {threads}
         """
 
+def _clock_rate_params(wildcards):
+    """
+    Generate the clock rate parameters for dengue samples for augur refine based on if wildcard serotype and gene values are in the config file
+
+    refine:
+      clock_rate:
+        wildcards.serotype:
+          wildcards.gene: numeric value here
+
+    else leave blank
+    """
+    clock_rate = config.get('refine', {}).get('clock_rate', {}).get(wildcards.serotype, {}).get(wildcards.gene, "")
+    if clock_rate !="":
```

Comment:
One way to override/remove the default config param is set the param to `null` in a custom config overlay. This check would not catch those cases. Since `null` converts to `None`, would be better to just check for `None` here

```suggestion
    clock_rate = config.get('refine', {}).get('clock_rate', {}).get(wildcards.serotype, {}).get(wildcards.gene, None)
    if clock_rate is not None:
```